intro.tex

```diff
@@ -0,0 +1,15 @@
+
+\section{Introduction}
+\label{sec:intro}
+
+The Verification Control Document (VCD) is a detailed look at where we are in verifying our requirements e.g. \citeds{LSE-61}.
+We verify requirements by
+running tests, the results of which are recorded in Jira.  A description of how the verification testing is set up is given in Section 2 of \citeds{LDM-503}.
+Specifically Section 2.3.2 of \citeds{LDM-503} describes the verification elements, test cases which appear in \secref{sec:vcd}.
```

Comment:
Comma after "Specifically".  Replace comma after "verification elements" with "and".